data/themes/darktable.css

```diff
@@ -1779,6 +1779,17 @@ scale contents trough slider:hover
   background-color: @scroll_bar_active;
 }
 
+.iop_drop_before
+{
+   border: 2px solid black;
+   border-bottom: 3.5em solid black;
+}
+
+.iop_drop_after
+{
+   border: 2px solid black;
+   border-top: 3.5em solid black;
+}
```

Comment:
3.5em is really too big and make that not fine-tuned. 2.6em is for me the better much better choice  as it's just little more bigger than module height. 2.8em is the higher acceptable value here for me. On both lines of course.

For all lines, color black is not suitable on all themes. I just suggest to replace them by @plugin_bg_color that is exactly done for such a new thing.

Reply:
> 2.6em is for me the better much better choice...2.8em is the higher acceptable value here for me.

Fine with me. I prefer 2.8 personally (or perhaps even a little bigger - see screenshot below).

> I just suggest to replace them by @plugin_bg_color

The borders serve two purposes here. Firstly to make a space to indicate where the dragged module will be dropped but secondly to highlight the destination module to which the dragged module is being "pinned" (i.e. to show which module it will be placed immediately before or after). Because either module may or may not be expanded, the border was the only way I could think of to indicate this. 

I could perhaps make the remainder of the border a bit larger. So, for example the following indicates that the exposure module will be dropped immediately before channel mixer (here the bottom border is 2.8em and the remaining borders are 4px):

![Screenshot_2020-07-20_20-54-58](https://user-images.githubusercontent.com/9555491/87980360-6337e200-cacb-11ea-9e12-c467dda77c04.png)

Open to suggestions of other ways to indicate this

Reply:
I understand all purposes but that's not good the way you propose and I don't understand the problem. With 2.8em and plugin_bg_color, on all themes, I see without any problem where I drag and drop a module (start point and end point). The way you propose with black and 3.5em make that just really too big and not adapted to the theme color, especially in grey and dark ones of course (for black color) and so plugin_bg_color just adapt that. On darker theme and so default one, plugin_bg_color is just a color near to black.

On screenshot you post, I don't see any problems, we see module moved and yes, the idea for me is to not have too much space around (quite awful and not consistent with the fine-tuned UI we made for 3.2). We see that we move module without needed of so much space and with 2.8, you still see a border around (a thin one and it's just sufficient for me). I find even 3em too big.

Reply:
Here it is with 2.8em and 2px (exposure will be placed after color mapping in the pipe)

![Screenshot_2020-07-20_21-13-20](https://user-images.githubusercontent.com/9555491/87981923-ebb78200-cacd-11ea-8d5b-05cea34dbcdd.png)

Ok I can live with 2px. I agree 4px is too much

Reply:
Thanks. All good for me and works great on my side.

Reply:
Excellent. Thanks @Nilvus

Reply:
Oh, even if it's not the subject here, I name you in the future 3.2 article for darktable.org/blog (and so translation versions), so if you want to check/read: https://github.com/darktable-org/dtorg/pull/111